core/00_primordials.js

```diff
@@ -31,508 +31,503 @@
-   * @returns A new Promise.
-   */
-  primordials.SafePromiseAll = (values) =>
-    // Wrapping on a new Promise is necessary to not expose the SafePromise
-    // prototype to user-land.
-    new Promise((a, b) =>
-      SafePromise.all(
-        ArrayPrototypeMap(
-          values,
-          (p) => {
-            if (ObjectPrototypeIsPrototypeOf(PromisePrototype, p)) {
-              return new SafePromise((c, d) => PromisePrototypeThen(p, c, d));
-            }
-            return p;
-          },
-        ),
-      ).then(a, b)
-    );
-
-  /**
-   * Attaches a callback that is invoked when the Promise is settled (fulfilled or
-   * rejected). The resolved value cannot be modified from the callback.
-   * Prefer using async functions when possible.
-   * @param {Promise<any>} thisPromise
-   * @param {() => void) | undefined | null} onFinally The callback to execute
-   *        when the Promise is settled (fulfilled or rejected).
-   * @returns A Promise for the completion of the callback.
-   */
-  primordials.SafePromisePrototypeFinally = (thisPromise, onFinally) =>
-    // Wrapping on a new Promise is necessary to not expose the SafePromise
-    // prototype to user-land.
-    new Promise((a, b) =>
-      new SafePromise((a, b) => PromisePrototypeThen(thisPromise, a, b))
-        .finally(onFinally)
-        .then(a, b)
-    );
-
-  // Create getter and setter for `queueMicrotask`, it hasn't been bound yet.
-  let queueMicrotask = undefined;
-  ObjectDefineProperty(primordials, "queueMicrotask", {
-    get() {
-      return queueMicrotask;
-    },
-  });
-  primordials.setQueueMicrotask = (value) => {
-    if (queueMicrotask !== undefined) {
-      throw new Error("queueMicrotask is already defined");
-    }
-    queueMicrotask = value;
-  };
+// Create getter and setter for `queueMicrotask`, it hasn't been bound yet.
+let queueMicrotask = undefined;
+ObjectDefineProperty(primordials, "queueMicrotask", {
+  get() {
+    return queueMicrotask;
+  },
+});
+primordials.setQueueMicrotask = (value) => {
+  if (queueMicrotask !== undefined) {
+    throw new Error("queueMicrotask is already defined");
+  }
+  queueMicrotask = value;
+};
 
-  // Renaming from `eval` is necessary because otherwise it would perform direct
-  // evaluation, allowing user-land access to local variables.
-  // This is because the identifier `eval` is somewhat treated as a keyword
-  primordials.indirectEval = eval;
+// Renaming from `eval` is necessary because otherwise it would perform direct
+// evaluation, allowing user-land access to local variables.
+// This is because the identifier `eval` is somewhat treated as a keyword
+primordials.indirectEval = eval;
 
-  ObjectSetPrototypeOf(primordials, null);
-  ObjectFreeze(primordials);
+ObjectSetPrototypeOf(primordials, null);
+ObjectFreeze(primordials);
 
-  // Provide bootstrap namespace
-  globalThis.__bootstrap = { primordials };
-})();
+export default primordials;
```

Comment:
In the limit, let's codemod this, and all the primordials import sites to individually import/export each primordial.

(for follow up)